CHANGELOG.md

```diff
@@ -1,3 +1,37 @@
+## v0.60.0 [2022-05-25]
+
+_What's new?_
+
+- Added `ignore_error_codes` config arg and update all tables to use `IgnoreConfig` instead of `ShouldIgnoreError` in `GetConfig`. ([#992](https://github.com/turbot/steampipe-plugin-aws/pull/992))
```

Comment:
```suggestion
- Added `ignore_error_codes` config arg to provide users the ability to set a list of additional AWS error codes to ignore while running queries. For instance, to ignore some common access denied errors, which is helpful when running with limited permissions, set the argument `ignore_error_codes = ["AccessDenied", "AccessDeniedException"]`. For more information, please see [AWS plugin configuration](https://hub.steampipe.io/plugins/turbot/aws#configuration). ([#992](https://github.com/turbot/steampipe-plugin-aws/pull/992))
```

Can you please also break out `Updated all tables to use ...` as a separate note to be under the Enhancements section?